src/clangsa/GCChecker.cpp

```diff
@@ -698,7 +708,7 @@ void GCChecker::checkEndFunction(const clang::ReturnStmt *RS,
   // If `RS` is NULL, the function did not return normally.
   // This could be either an abort/assertion failure or an exception throw.
   // Do not require the GC frame to match in such case.
-  if (RS && C.getState()->get<GCDepth>() > 0)
```

Comment:
Also remove the comment or update it to clarify what `NULL` `RS` actually mean? AFAICT it does corresponds to ending the function from non-return but I guess somehow the core analysis deal with it so that the control flow is handled correctly?

Reply:
Null `RS` can also mean that there was no return statement in this function, which is a fine thing to do in C.